tf-psa-crypto/framework/scripts/project_detection.sh

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env sh
+# project-detection.sh
```

Comment:
I think this can be avoided, see https://github.com/Mbed-TLS/mbedtls/pull/9763/files#r1837760974